src/components/Collections/Collection.tsx

```diff
@@ -260,29 +266,31 @@ export function Collection({
                           <IconDotsVertical size={16} />
                         </ActionIcon>
                       </Menu.Target>
-                      <Menu.Dropdown>
-                        {/*// TODO.PersonalizedHomePages: This is disabled for now until fully*/}
-                        {/*implemented*/}
-                        {/*<Menu.Item*/}
-                        {/*  icon={<IconHome size={14} stroke={1.5} />}*/}
-                        {/*  onClick={(e) => {*/}
-                        {/*    e.preventDefault();*/}
-                        {/*    e.stopPropagation();*/}
-                        {/*    onToggleCollectionHomeBlock();*/}
-                        {/*  }}*/}
-                        {/*>*/}
-                        {/*  {collectionHomeBlock ? 'Remove from my home' : 'Add to my home'}*/}
-                        {/*</Menu.Item>*/}
-                        {permissions.manage && (
-                          <Menu.Item
-                            component={NextLink}
-                            icon={<IconPencil size={14} stroke={1.5} />}
-                            href={`/collections/${collection.id}/review`}
-                          >
-                            Review Items
-                          </Menu.Item>
-                        )}
-                      </Menu.Dropdown>
+                      {(permissions.read || permissions.manage) && (
```

Comment:
I guess it'd make sense to put this check in line 262 to prevent showing the action icon if the user doesn't have the permissions